config.json

```diff
@@ -1,6 +1,6 @@
 {
   "port":8080,
-  "broker_hosts":["localhost:9092"],
+  "broker_hosts":["as01.hadoop:9092"],
```

Comment:
This must have slipped in during your commit. Better change it back to localhost.